pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_guidance-on-using-xrefs.adoc

```diff
@@ -0,0 +1,55 @@
+[id="guidance-on-using-xrefs_{context}"]
+= Guidance on using xrefs
+
+[role="_abstract"]
+To ensure that internal cross-references, xrefs, work when you publish your content, use only inter-document xrefs. Although AsciiDoc supports different forms of xref, {ProductShortName} validates only inter-document xrefs. If validation fails, {ProductShortName} flags invalid xrefs.
+
+NOTE: {ProductShortName} only validates xrefs within a repository.
+
+The following are the syntaxes of inter-document xref:
+
+* `\xref:document-b.adoc#target-section[Target section]`
+* `\xref:../modules/document-c.adoc[Content C]`
+
+[discrete]
+== Example of using xrefs
+
+In this example, the following are the source and targets:
+
+source document:: 
+* A file `document-a.adoc`.
+
+Targets:: 
+
+* A section called _target section_ in the file _document-b.adoc_.
+* A file _document-c.adoc_.
+
+Use xrefs as follows:
+
+* To cross-reference section target section in document-b.adoc from document-a.adoc, use the following form:
+
+** `\xref:document-b.adoc#target-section[target section]`
++
+where `target-section` is the anchor for _target section_.
+
+* To cross-reference document document-c.adoc from document-a.adoc, use the following form:
+
+** `\xref:../modules/document-c.adoc[Section C]`
++
+where `../modules/document-c.adoc` is the path to _document-c.adoc_ relative to _document-a.adoc_.
+
+IMPORTANT: Always include text within the square brackets in an xref.
```

Comment:
```suggestion
[IMPORTANT]
====
Always include text within the square brackets in an xref.
====
```
Is this a Pantheon requirement?

Reply:
yes, it's documented in the pv2 migration checklist